pkg/sfu/streamtrackermanager.go

```diff
@@ -35,32 +35,30 @@ type StreamTrackerManager struct {
 	onMaxLayerChanged            func(maxLayer int32)
 }
 
-func NewStreamTrackerManager(logger logger.Logger, trackInfo *livekit.TrackInfo, isSVC bool, trackerConfig config.StreamTrackersConfig) *StreamTrackerManager {
+func NewStreamTrackerManager(
+	logger logger.Logger,
+	trackInfo *livekit.TrackInfo,
+	isSVC bool,
+	clockRate uint32,
+	trackerType config.StreamTrackerType,
+	trackersConfig config.StreamTrackersConfig,
+) *StreamTrackerManager {
 	s := &StreamTrackerManager{
-		logger:                    logger,
-		trackInfo:                 trackInfo,
-		isSVC:                     isSVC,
-		maxPublishedLayer:         0,
-		exemptedLayersVideo:       trackerConfig.ExemptedLayersVideo,
-		exemptedLayersScreenshare: trackerConfig.ExemptedLayersScreenshare,
+		logger:            logger,
+		trackInfo:         trackInfo,
+		isSVC:             isSVC,
+		maxPublishedLayer: 0,
+		clockRate:         clockRate,
+		trackerType:       trackerType,
 	}
 
-	for _, layer := range trackerConfig.Video {
-		s.configVideo = append(s.configVideo, StreamTrackerParams{
-			SamplesRequired:       layer.SamplesRequired,
-			CyclesRequired:        layer.CyclesRequired,
-			CycleDuration:         layer.CycleDuration,
-			BitrateReportInterval: layer.BitrateReportInterval,
-		})
-	}
-
-	for _, layer := range trackerConfig.Screenshare {
-		s.configScreenshare = append(s.configScreenshare, StreamTrackerParams{
-			SamplesRequired:       layer.SamplesRequired,
-			CyclesRequired:        layer.CyclesRequired,
-			CycleDuration:         layer.CycleDuration,
-			BitrateReportInterval: layer.BitrateReportInterval,
-		})
+	switch s.trackInfo.Source {
+	case livekit.TrackSource_SCREEN_SHARE:
+		s.trackerConfig = trackersConfig.Screenshare
+	case livekit.TrackSource_CAMERA:
+		s.trackerConfig = trackersConfig.Video
+	default:
```

Comment:
Using `Video` settings by default as some of the tests don't have `TrackSource`.